### DISCUSSION TITLE
支持全台最好的媒體

### DISCUSSION THREAD

**Discussion by Chattenoire** [1101-GS4719-C/Python-Community-GS4719-C#2216] [Ideas]
如題，想必在這個時代大家應該都知道報導者了，(理想上)應該沒有人會繼續看甚麼東森每日新聞還是什麼其他的垃圾了吧....

https://www.twreporter.org/

台灣媒體會如此不堪，一半是記者本身素質問題，一半是觀眾自己只喜歡低俗的東西。
所以，各位來改變一下環境吧，就從停止觀看沒用的媒體內容開始。

**Comment by 112121py** [comment]
關於台灣媒體的亂象，之前有一位youtuber 叫錫蘭，他也曾經受過台灣媒體的「荼毒」，後來有做一部關於台灣媒體亂象的影片，影片很長，可是我覺得他分析的非常好，點出很多存在很久的問題，只是因為環境，沒有被解決，而是一直存在，可以去看看～
[台灣媒體的下限能有多低？](https://youtu.be/3tfy3yMnS78?si=rqhDsM8bZsU8umfg)

**Comment by Jinyuu17** [comment]
在現今社會中，媒體識讀能力很重要，假訊息充斥著我們的生活，判斷訊息的正確性比選擇收看哪家媒體還重要。

**Comment by Jinyuu17** [comment]
其實之前有做過研究，台灣人覺得可信的媒體跟台灣人會看的媒體相關性非常的低。這邊推薦一些我整理背景單純可靠的媒體： 公視新聞、商業週刊、天下雜誌

另外，今日新聞有不要再看了，內容充斥著對岸的想法和農場文章。

**Comment by yuan3x2** [comment]
我自己是喜歡看cnn或bbc順便練習英文，還可以知道一些他國的新聞，撇除政治，關於台灣媒體的部分我有看到過別人的評論是說，許多諸如車禍等等的事情幾乎每天都在發生，因此類似鮭魚之亂或獵人的新聞才會吸引到相對多的收視率，再加上台灣相對小，總之就是一個蠻有趣的現象。

**Comment by DarsooK** [comment]
台灣媒體亂象主要指台灣於1980年代解嚴後，由於商業媒體（特指新聞媒體）利用暴力、血腥、煽情、炒作與捏造新聞等違反媒體倫理的手段，增加收視率或報刊發行量之現象。商業媒體背後都由財團控制，政治傾向影響真實信與播報品質。

**Reply by zoo100130** [reply]
我都在半夜看公視的<我們的島>，用紀錄片形式講述一些更值得社會大眾的議題，內容的安排上也不會很無聊，個人認為在深度、娛樂性上都有很舒適的觀賞體驗，影片長度也不會太長，大概一集20幾分鐘。
